pennylane_qrack/qrack_device.py

```diff
@@ -200,6 +202,7 @@ def __init__(self, wires=0, shots=None, **kwargs):
             isHostPointer=self.isHostPointer,
             noise = self.noise
         )
+        self._circuit = []
```

Comment:
If this is defined here as an empty list, does the manipulation of the variable below make sense? That is, can we just set `self._circuit = operations` below or am I missing something?

Reply:
You _are_ missing something: `apply()` might be called multiple times before `generate_samples()` is called. Hence, it needs to append, not replace the entire set of operations up to that point.